static/app/components/devtoolbar/components/feedback/feedbackPanel.tsx

```diff
@@ -120,71 +121,67 @@ function FeedbackListItem({item}: {item: FeedbackIssueListItem}) {
   const hasComments = item.numComments > 0;
 
   return (
-    <div css={listItemGridCss}>
-      <TextOverflow css={smallCss} style={{gridArea: 'name'}}>
-        <SentryAppLink
-          to={{
-            url: '/feedback/',
-            query: {project: projectId, feedbackSlug: `${projectSlug}:${item.id}`},
-          }}
-          onClick={() => {
-            trackAnalytics?.({
-              eventKey: `devtoolbar.feedback-list.item.click`,
-              eventName: `devtoolbar: Click feedback-list item`,
-            });
-          }}
-        >
-          <strong>
-            {item.metadata.name ?? item.metadata.contact_email ?? 'Anonymous User'}
-          </strong>
-        </SentryAppLink>
-      </TextOverflow>
-
-      <div
-        css={[gridFlexEndCss, xSmallCss]}
-        style={{gridArea: 'time', color: 'var(--gray300)'}}
-      >
-        <TimeSince date={item.firstSeen} unitStyle="extraShort" />
-      </div>
-
-      <div style={{gridArea: 'message'}}>
-        <TextOverflow css={[smallCss, textOverflowTwoLinesCss]}>
-          {item.metadata.message}
+    <AnalyticsProvider nameVal="item" keyVal="item">
```

Comment:
<img width="521" alt="SCR-20240730-mfhx" src="https://github.com/user-attachments/assets/06086bb4-28f1-4a2d-9dde-527b17808398">

Reply:
added to amplitude